Rakefile

```diff
@@ -80,10 +80,16 @@ task :swift_lint_update do
   File.write(File.join(root_dir, "vendor/.swiftlint.version"), SWIFTLINT_VERSION)
 end
 
+desc("Install git hooks")
```

Comment:
We introduced a replacement for Rakefile, `fourier`, with the goal of having a better CLI experience interacting with the project. That being said, I'd make this part of a new command:

```
fourier setup
```

Reply:
sure, I'll move it to `fourier`